api-server/server/boot/authentication.js

```diff
@@ -25,15 +29,34 @@ module.exports = function enableAuthentication(app) {
   // loopback.io/doc/en/lb2/Authentication-authorization-and-permissions.html
   app.enableAuth();
   const ifUserRedirect = ifUserRedirectTo();
+  const saveAuthCookies = saveResponseAuthCookies();
+  const loginSuccessRedirect = loginRedirect();
   const ifNoUserRedirectHome = ifNoUserRedirectTo(homeLocation);
   const api = app.loopback.Router();
   const { AuthToken, User } = app.models;
 
-  api.get('/signin', ifUserRedirect, passport.authenticate('auth0-login', {}));
-  api.get(
-    '/auth/auth0/callback',
-    createPassportCallbackAuthenticator('auth0-login', { provider: 'auth0' })
-  );
+  // Use a local mock strategy for signing in if we are in dev mode.
+  // Otherwise we use auth0 login. We use a string for 'true' because values
+  // set in the env file will always be strings and never boolean.
+  if (process.env.LOCAL_AUTH === 'true') {
```

Comment:
```suggestion
  if (process.env.LOCAL_MOCK_AUTH === 'true') {
```

Sorry about the nit-pick. Let's make it a little bit less generic, and avoid potential conflict in future? Not that I am aware of that there is anything in the code or underlying libraries would use `LOCAL_AUTH`, it does help to be specific, while we are at it?

Reply:
The reason I used `LOCAL_AUTH` is that you already have it in the `sample.env`. Do you want to go with `LOCAL_MOCK_AUTH` and I'll update the `sample.env` too?